backend/action_logs/views.py

```
@@ -235,43 +316,145 @@ def reject(self, request, pk=None):
+        action_log = self.get_object()
+        ActionLogNotification.objects.filter(
+            action_log=action_log,
+            user=request.user,
+            is_read=False
+        ).update(is_read=True)
+        return Response({'marked_read': True})
+
+class ActionLogCommentViewSet(viewsets.ModelViewSet):
+    queryset = ActionLogComment.objects.all()
+    serializer_class = ActionLogCommentSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)